CONTRIBUTING.md

```diff
@@ -21,7 +21,7 @@ We use GitHub as our main development platform. GitHub features a "Pull Request"
  5. Members of the MEI Technical Team will then review your submission. If you are asked to make changes, you can push these changes to your original branch and the pull request will be automatically updated.
  6. Once the changes have gone through our review process, a Core Contributor will merge your changes into the MEI source repository. You may then delete your dedicated branch.
 
-## Checklist for a contribution
+## Checklist for a Significant Pull Request
```

Comment:
It does not have to be significant. Actually, I would say the contrary. Significant PRs need preliminary discussion.

Reply:
I was just trying to make the heading of this section agree with the contents, which is entirely about "significant" changes! But I'm happy to leave that as is and let someone else fix it if they want.

Reply:
I agree with @donbyrd 's motivation to encourage more people to engage in the development process, without necessarily making a pull request.  How about changing --
```
We use GitHub as our main development platform. GitHub features a 
"Pull Request" mechanism for proposing contributions to the schema 
from our community. A brief overview of this process:
```
to
```
We use GitHub as our main development platform. An easy, non-technical
way of contributing to the development of MEI is to create a new issue or 
respond to an existing one.  For the more technically-inclined, GitHub features 
a "Pull Request" mechanism for proposing modifications to the schema code. 
A brief overview of this process:
```
and
```
Checklist for a contribution
```
to
```
Checklist for a schema contribution
```

Reply:
Perfect! That's what I had in mind, but your wording is much better, @pe-ro .

Reply:
@donbyrd, Can you make these changes please?